### ISSUE TITLE
Deleting scripts isn't permanent

### ISSUE THREAD

**Issue by jeandrek** [LLK/scratch-vm#140]
If I delete the script in the stage in the default project, then it still runs, even if I press Stop and then Green Flag again, and if I go to a sprite and then back to the stage, the script reappears.

**Comment by tmickel** [comment]
Not 100% sure but I think this is a side effect of the bugs from #122. The references are likely a little off in your blocks, and the delete is probably throwing an error (can you give us the console output?)

If so, should be fixed with that PR!

**Comment by jeandrek** [comment]
I get `TypeError: block is undefined` when I delete it. Then if I press Green Flag, I get lots of

```
uncaught exception: Tried to glow block that does not exist.
uncaught exception: Tried to glow stack on block that does not exist.
uncaught exception: Tried to glow block that does not exist.
uncaught exception: Tried to glow stack on block that does not exist.
```

until I go to a sprite and then maybe back to the stage.

**Comment by tmickel** [comment]
@Jonathan50: I think this is resolved in the latest updates, minus #91. Please reopen if you experience it again!

**Comment by jeandrek** [comment]
It works fine now.
